src/react/components/common/keyboardBinding/keyboardBinding.tsx

```diff
@@ -14,11 +24,7 @@ export class KeyboardBinding extends React.Component<IKeyboardBindingProps> {
 
     public componentDidMount() {
         if (this.context && this.context.keyboard) {
-            this.deregisterBinding = this.context.keyboard.addHandler(
-                this.props.keyEventType || KeyEventType.KeyDown,
-                this.props.accelerators,
-                this.props.onKeyEvent,
-            );
+            this.deregisterBinding = this.context.keyboard.registerBinding(this.props);
```

Comment:
It is a bit weird to think about, but I actually think it's pretty slick (Wallace's idea, not mine). But specifically for the help menu, it's awesome because it allows you to only show bindings that are relevant for the context that you are in